database/migrations/20210317210236-geography.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function (db, callback) {
+  db.runSql(
```

Comment:
calling the callback twice like this won't give you the behavior you desire.  It will work better to create 2 separate migration files, once for each column you are adding.

Reply:
Thanks for the feedback!